README.md

```diff
@@ -48,7 +48,7 @@ PeekingDuck is an open-source, modular framework in Python, built for Computer V
 
     <img src="https://raw.githubusercontent.com/aimakerspace/PeekingDuck/dev/images/readme/yolo_demo.gif" width="50%">
 
-    The previous command looks for a `run_config.yml` in the current directory. You can also specify the path of a different config file to be used, as follows:
+    The previous command looks for a `pipeline.yml` in the current directory. You can also specify the path of a different config file to be used, as follows:
     ```
     > peekingduck run --config_path <path_to_config>
```

Comment:
`path_to_pipeline` ?

should `--config_path` remain as is or be changed to `pipeline_path` too?

Reply:
I am ok to leave `config_path` for backward compatibility, and also since it's dealing with "pipeline configuration".